Answers.md

```diff
@@ -1,3 +1,15 @@
 ### Your responses to the short answer questions should be laid out here using markdown.
 
 For help with markdown syntax, [go here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+
+1. A virtual machine is an OS running on top of an OS in a virtual environment. Windows running on a Mac whilst having OSX as the main OS, is just one example. Docker creates a copy of the dev environment
```

Comment:
Docker also doesn't contain an entire copy of the operating system within the container.